alembic/env.py

```diff
@@ -1,13 +1,13 @@
 import asyncio
 from logging.config import fileConfig
 
-from alembic import context
 from sqlalchemy import engine_from_config, pool
 from sqlalchemy.ext.asyncio import AsyncEngine
 from sqlalchemy.ext.asyncio.engine import AsyncConnection
 
 # This is a required step by Alembic to properly generate migrations
 import api.core.database.models  # noqa: F401
+from alembic import context
```

Comment:
this is incorrect, as alembic is a 3rd party lib, so shouldn't be grouped with 1st party packages.

I imagine black is doing this since it seems there is an `alembic` folder in the project, so assumes all references to `alembic` are 1st party.

We should rename the folder so that we don't clobber the namespace.

Reply:
> I imagine black is doing this since it seems there is an alembic folder in the project, so assumes all references to alembic are 1st party.

A: black doesn't sort packages, isort does.
B: There's a configuration option for this.
```toml
[tool.isort]
known_third_party = ["alembic"]
```
https://pycqa.github.io/isort/docs/configuration/options.html#known-third-party